_migration-assistant/migration-phases/planning-your-migration/handling-type-mapping-deprecation.md

```diff
@@ -0,0 +1,189 @@
+
+[^1]: The `/shared-logs-output` mount will soon be relocated with a new mountpoint on the container for the EFS volume.
+
+### Configuration Options
+
+The TypeMappingsSanitizationTransformer supports several strategies for handling type mappings:
+
+1. **Route to Separate Indices**: Split different types into their own indices
+2. **Merge Types**: Combine multiple types into a single index
+3. **Drop Types**: Selectively migrate only specific types
+4. **Keep Original Structure**: Maintain the same index name while conforming to new type standards
+
+### Example Configurations
+
+Here are some common scenarios and their corresponding configurations:
+
+#### 1. Route Different Types to Separate Indices
+
+If you have an index `activity` with types `user` and `post` that you want to split into separate indices:
+
+```json
+[
+  {
+    "TypeMappingsSanitizationTransformerProvider": {
+      "staticMappings": {
+        "activity": {
+          "user": "new_users",
+          "post": "new_posts"
+        }
+      },
+      "sourceProperties": {
+        "version": {
+          "major": 6,
+          "minor": 8
+        }
+      }
+    }
+  }
+]
+```
+
+This will:
+- Route documents with type `user` to the `new_users` index
+- Route documents with type `post` to the `new_posts` index
+
+#### 2. Merge All Types into One Index
+
+To combine multiple types into a single index:
+
+```json
+[
+  {
+    "TypeMappingsSanitizationTransformerProvider": {
+      "staticMappings": {
+        "activity": {
+          "user": "combined_activity",
+          "post": "combined_activity"
+        }
+      },
+      "sourceProperties": {
+        "version": {
+          "major": 6,
+          "minor": 8
+        }
+      }
+    }
+  }
+]
+```
+
+#### 3. Drop Specific Types
+
+To migrate only specific types and drop others:
+
+```json
+[
+  {
+    "TypeMappingsSanitizationTransformerProvider": {
+      "staticMappings": {
+        "activity": {
+          "user": "users_only",
+        }
+      },
+      "sourceProperties": {
+        "version": {
+          "major": 6,
+          "minor": 8
+        }
+      }
+    }
+  }
+]
+```
+This configuration will only migrate documents of type `user` and ignore other types in the `activity` index.
+
+### 4. Keep original structure
+
+To migrate only specific types and keep the original structure:
+
+```json
+[
+  {
+    "TypeMappingsSanitizationTransformerProvider": {
+      "regexMappings": [
+        [
+          "(.*)",
+          ".*",
+          "$1"
+        ]
+      ],
+      "sourceProperties": {
+        "version": {
+          "major": 6,
+          "minor": 8
+        }
+      }
+    }
+  }
+]
+```
+
+This is equivalent to the merge all types into one index rule but using a pattern-based routing strategy.
+
+### Defaults
+
+When no regex mappings are specified, the transformer will default to the following behavior:
```

Comment:
Is this when nothing at all is specified? Or `regexMappings` is specified but it's empty?